### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by mwmsh** [mwmsh/linux#1]
@ellipsis-dev does this codebase contain a readme?

**Comment by mwmsh** [comment]
@Ellipsis does this codebase contain a README file?

**Comment by mwmsh** [comment]
@ellipsis-dev does this codebase contain a readme?

**Comment by hbrooks** [comment]
Hey @mwmsh sorry about this. 

Ellipsis can answer questions on Pull Requests, but not on GitHub issues at the moment. It can only create pull requests from issues.

**Comment by mwmsh** [comment]
> Hey @mwmsh sorry about this.
> 
> Ellipsis can answer questions on Pull Requests, but not on GitHub issues at the moment. It can only create pull requests from issues.

Thanks @hbrooks  for the update. Is ellipsis available for anything beyond code reviews? Can it answer general questions about a codebase?

**Comment by hbrooks** [comment]
@mwmsh Yes, you can use Slack to ask Q&A over the entire codebase, but we don't have self service for that yet though - I'm happy to manually onboard you. Can we schedule time? https://cal.com/hunter-brooks/15min

**Comment by mwmsh** [comment]
@ellipsis-dev does this codebase contain a readme?

**Comment by mwmsh** [comment]
@ellipsis-dev does this codebase contain a readme?
